run.py

```diff
@@ -19,6 +19,16 @@
 # Get YT url
 vidUrl = sys.argv[1]
 
+# Testing flag ... will not update db if PROD = FALSE
+PROD = False
```

Comment:
https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5

I think this could be a better way for setting our `test` or `prod` env vars. If we could have an env var created locally that would control this behaviour. That way we don't need to pass it in the arguments

Reply:
actually this is really nice and would solve our secrets.py mess too!